unison-src/transcripts/find-command.md

```diff
@@ -14,37 +13,36 @@ somewhere.bar = 7
 ```
 
 ```ucm:hide
-.> add
+scratch/main> add
 ```
 
 ```ucm
-.> find foo
-.> view 1
-.> find.all foo
-.> view 1
+scratch/main> find foo
+scratch/main> view 1
+scratch/main> find.all foo
+scratch/main> view 1
 ```
 
 ```ucm
-.> find-in cat foo
-.> view 1
-.> find-in.all cat foo
-.> view 1
+scratch/main> find-in cat foo
+scratch/main> view 1
+scratch/main> find-in.all cat foo
+scratch/main> view 1
 ```
 
-```ucm
-.somewhere> find bar
-.somewhere> find.global bar
-```
+Finding within a namespace
 
 ```ucm
-.> find bar
-.> find-in somewhere bar
+scratch/main> find bar
+-- Shows UUIDs
+-- scratch/main> find.global bar
```

Comment:
This shows project UUIDs unfortunately, I'm hoping I can revive this behaviour correctly in the project-roots branch